### ISSUE TITLE
Validation set upload

### ISSUE THREAD

**Issue by CatOneTwo** [ucla-mobility/V2V4Real#8]
I noticed that the train and test set of V2V4Real has been uploaded, but the validation set is still lacking. Could you upload the validation set? Thank you~

**Comment by DerrickXuNu** [comment]
Hi, actually we are thinking of holding that validation part for some future competition. So please just use train and test for now

**Comment by CatOneTwo** [comment]
I see, thanks～
